README.md

```diff
@@ -1,8 +1,6 @@
 # OWASP Domain Protect
-<!--
-[![Version](https://img.shields.io/github/v/release/domain-protect/terraform-aws-domain-protect)](https://github.com/domain-protect/terraform-aws-domain-protect/releases/tag/0.1.0)
--->
 
+[![Version](https://img.shields.io/github/v/release/domain-protect/terraform-aws-domain-protect)](https://github.com/domain-protect/terraform-aws-domain-protect/releases/tag/0.5.0)
```

Comment:
Is there a way to always show the latest?

Reply:
This will always show the latest version on the README, however if you were to click into the version image (which you would never normally do), then it would link to an older location.
So this won't need to be updated - see the [Serverless CA project](https://github.com/serverless-ca/terraform-aws-ca) as an example of how this works.

Reply:
Can we just omit the tag so it goes to the main releases page ?

```suggestion
[![Version](https://img.shields.io/github/v/release/domain-protect/terraform-aws-domain-protect)](https://github.com/domain-protect/terraform-aws-domain-protect/releases)
```